third-party/CMakeLists.txt

```diff
@@ -44,8 +44,7 @@ list(APPEND EXTRA_INCLUDE_PATHS
   "${TP_DIR}/forks/libmbfl/filters"
 )
 
-list(APPEND THIRD_PARTY_MODULES forks/timelib)
-list(APPEND EXTRA_INCLUDE_PATHS "${TP_DIR}/forks/timelib")
+list(APPEND THIRD_PARTY_MODULES timelib)
```

Comment:
note that for now, `third-party/forks/timelib` will still exist and just won't be used. we can clean it up later in the hhvm-third-party repo I suppose?